# adventr

<details>

* Version: 0.1.8
* Source code: https://github.com/cran/adventr
* Date/Publication: 2020-05-05 16:50:06 UTC
* Number of recursive dependencies: 162

Run `revdep_details(,"adventr")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        tutorials   5.5Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BayesFactor’ ‘GGally’ ‘Hmisc’ ‘WRS2’ ‘boot’ ‘car’ ‘dplyr’ ‘effects’
      ‘effsize’ ‘forcats’ ‘ggplot2’ ‘lm.beta’ ‘nlme’ ‘readr’ ‘robust’
      ‘sandwich’ ‘sjstats’ ‘tidyr’
      All declared Imports should be used.
    ```

# alevinQC

<details>

* Version: 1.4.0
* Source code: https://github.com/cran/alevinQC
* URL: https://github.com/csoneson/alevinQC
* BugReports: https://github.com/csoneson/alevinQC/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 87

Run `revdep_details(,"alevinQC")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 12.7Mb
      sub-directories of 1Mb or more:
        extdata  11.6Mb
    ```

# ALPS

<details>

* Version: 1.2.0
* Source code: https://github.com/cran/ALPS
* URL: https://github.com/itsvenu/ALPS
* BugReports: https://github.com/itsvenu/ALPS/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 194

Run `revdep_details(,"ALPS")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# BGGM

<details>

* Version: 2.0.0
* Source code: https://github.com/cran/BGGM
* BugReports: https://github.com/donaldRwilliams/BGGM/issues
* Date/Publication: 2020-05-31 21:10:09 UTC
* Number of recursive dependencies: 185

Run `revdep_details(,"BGGM")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        doc   2.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘RcppProgress’
      All declared Imports should be used.
    ```

# bigPint

<details>

* Version: 1.4.0
* Source code: https://github.com/cran/bigPint
* URL: https://github.com/lindsayrutter/bigPint
* BugReports: https://github.com/lindsayrutter/bigPint/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 165

Run `revdep_details(,"bigPint")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  8.1Mb
      sub-directories of 1Mb or more:
        data             2.1Mb
        doc              2.7Mb
        shiny-examples   3.0Mb
    ```

# CeTF

<details>

* Version: 1.0.2
* Source code: https://github.com/cran/CeTF
* Date/Publication: 2020-05-31
* Number of recursive dependencies: 224

Run `revdep_details(,"CeTF")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        data   1.6Mb
        doc    3.0Mb
    ```

# CINNA

<details>

* Version: 1.1.53
* Source code: https://github.com/cran/CINNA
* Date/Publication: 2019-02-25 12:50:14 UTC
* Number of recursive dependencies: 136

Run `revdep_details(,"CINNA")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘circlize’
      All declared Imports should be used.
    ```

# CluMSID

<details>

* Version: 1.4.0
* Source code: https://github.com/cran/CluMSID
* URL: https://github.com/tdepke/CluMSID
* BugReports: https://github.com/tdepke/CluMSID/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 164

Run `revdep_details(,"CluMSID")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘xcms’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.2Mb
      sub-directories of 1Mb or more:
        doc   5.6Mb
    ```

# ctsem

<details>

* Version: 3.2.1
* Source code: https://github.com/cran/ctsem
* URL: https://github.com/cdriveraus/ctsem
* Date/Publication: 2020-04-26 01:00:07 UTC
* Number of recursive dependencies: 128

Run `revdep_details(,"ctsem")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.6Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        libs   6.3Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘numDeriv’ ‘pkgbuild’
      All declared Imports should be used.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# fic

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/fic
* URL: https://github.com/chjackson/fic
* BugReports: https://github.com/chjackson/fic/issues
* Date/Publication: 2019-04-13 08:32:39 UTC
* Number of recursive dependencies: 88

Run `revdep_details(,"fic")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘numDeriv’
      All declared Imports should be used.
    ```

# finalfit

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/finalfit
* URL: https://github.com/ewenharrison/finalfit
* BugReports: https://github.com/ewenharrison/finalfit/issues
* Date/Publication: 2020-04-21 11:50:02 UTC
* Number of recursive dependencies: 134

Run `revdep_details(,"finalfit")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
      mort_5yr mort_5yr.num   n
    1    Alive            1 511
    2     Died            2 404
    3     <NA>           NA  14
    
    $counts[[19]]
      sex.factor2 age.factor2   n
    1           M   <60 years 204
    2           M   60+ years 241
    3           F   <60 years 210
    4           F   60+ years 274
    
    
    > 
    > # Select a tibble and expand
    > out$counts[[9]] %>%
    +   print(n = Inf)
    Error in print.default(m, ..., quote = quote, right = right, max = max) : 
      invalid 'na.print' specification
    Calls: %>% ... print -> print.data.frame -> print -> print.default
    Execution halted
    ```

# fingerPro

<details>

* Version: 1.1
* Source code: https://github.com/cran/fingerPro
* URL: https://github.com/eead-csic-eesa
* Date/Publication: 2018-08-28 10:04:54 UTC
* Number of recursive dependencies: 154

Run `revdep_details(,"fingerPro")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘RcppProgress’
      All declared Imports should be used.
    ```

# GENESIS

<details>

* Version: 2.18.0
* Source code: https://github.com/cran/GENESIS
* URL: https://github.com/UW-GAC/GENESIS
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 150

Run `revdep_details(,"GENESIS")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.9Mb
      sub-directories of 1Mb or more:
        doc       3.2Mb
        extdata   3.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    ':::' calls which should be '::':
      ‘SPAtest:::Saddle_Prob’ ‘SPAtest:::Saddle_Prob_fast’
      See the note in ?`:::` about the use of this operator.
    Unexported object imported by a ':::' call: ‘survey:::saddle’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .pcrelate: no visible binding for global variable ‘k’
    calcISAFBeta: no visible binding for global variable ‘k’
    pcrelateSampBlock: no visible binding for global variable ‘k’
    Undefined global functions or variables:
      k
    ```

# ggbio

<details>

* Version: 1.36.0
* Source code: https://github.com/cran/ggbio
* URL: http://tengfei.github.com/ggbio/
* BugReports: https://github.com/tengfei/ggbio/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 157

Run `revdep_details(,"ggbio")` for more info

</details>

## In both

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'GenenameFilter' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

*   checking dependencies in R code ... NOTE
    ```
    ':::' call which should be '::': 'ggplot2:::set_last_plot'
      See the note in ?`:::` about the use of this operator.
    Unexported objects imported by ':::' calls:
      'S4Vectors:::top_prenv' 'ggplot2:::add_ggplot' 'ggplot2:::cunion'
      'ggplot2:::rename_aes' 'ggplot2:::rescale01'
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    layout_karyogram,GRanges: no visible binding for global variable 'xend'
    layout_karyogram,GRanges: no visible binding for global variable 'yend'
    layout_karyogram,GRanges: no visible binding for global variable 'y2'
    layout_karyogram,GRanges: no visible binding for global variable
      'yend2'
    layout_karyogram,GRanges: no visible binding for global variable 'name'
    plotFragLength,character-GRanges: no visible binding for global
      variable '.fragLength'
    plotSpliceSum,character-EnsDb: possible error in GRangesFilter(which,
      condition = "overlapping"): unused argument (condition =
      "overlapping")
    stat_mismatch,GRanges: no visible binding for global variable 'sts'
    stat_mismatch,GRanges: no visible binding for global variable 'eds'
    stat_mismatch,GRanges: no visible binding for global variable 'read'
    Undefined global functions or variables:
      .fragLength .layout_circle.stats .x breaks coefs data eds fe fl
      gieStain ideoCyto indexProbesProcessed midpoint mt name read se
      stepping sts value variable x xend y y.text y2 yend yend2
    Consider adding
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

# GGPA

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/GGPA
* URL: https://github.com/dongjunchung/GGPA/
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 73

Run `revdep_details(,"GGPA")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘network’ ‘scales’ ‘sna’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking compiled code ... NOTE
    ```
    File ‘GGPA/libs/GGPA.so’:
      Found ‘__ZNSt3__14coutE’, possibly from ‘std::cout’ (C++)
        Object: ‘3_Param.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# ggquickeda

<details>

* Version: 0.1.6
* Source code: https://github.com/cran/ggquickeda
* URL: https://github.com/smouksassi/ggquickeda
* BugReports: https://github.com/smouksassi/ggquickeda/issues
* Date/Publication: 2020-04-17 14:00:03 UTC
* Number of recursive dependencies: 157

Run `revdep_details(,"ggquickeda")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘Formula’ ‘GGally’ ‘Hmisc’ ‘colourpicker’ ‘dplyr’ ‘ggpmisc’
      ‘ggpubr’ ‘ggrepel’ ‘ggstance’ ‘grDevices’ ‘grid’ ‘gridExtra’
      ‘lazyeval’ ‘markdown’ ‘plotly’ ‘quantreg’ ‘rlang’ ‘shinyjqui’
      ‘shinyjs’ ‘survival’ ‘survminer’ ‘table1’ ‘tidyr’
      All declared Imports should be used.
    ```

# httk

<details>

* Version: 2.0.1
* Source code: https://github.com/cran/httk
* URL: https://www.epa.gov/chemical-research/rapid-chemical-exposure-and-dose-research
* BugReports: https://github.com/USEPA/CompTox-ExpoCast-httk
* Date/Publication: 2020-03-02 06:00:12 UTC
* Number of recursive dependencies: 111

Run `revdep_details(,"httk")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 12.7Mb
      sub-directories of 1Mb or more:
        data   9.1Mb
        doc    2.1Mb
    ```

# ICtest

<details>

* Version: 0.3-2
* Source code: https://github.com/cran/ICtest
* Date/Publication: 2019-08-23 11:50:02 UTC
* Number of recursive dependencies: 80

Run `revdep_details(,"ICtest")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘fICA’
    ```

# isomiRs

<details>

* Version: 1.16.2
* Source code: https://github.com/cran/isomiRs
* BugReports: https://github.com/lpantano/isomiRs/issues
* Date/Publication: 2020-06-03
* Number of recursive dependencies: 149

Run `revdep_details(,"isomiRs")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘isomiRs-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: mirna2targetscan
    > ### Title: Find targets in targetscan database
    > ### Aliases: mirna2targetscan
    > 
    > ### ** Examples
    > 
    > library(targetscan.Hs.eg.db)
    Error in library(targetscan.Hs.eg.db) : 
      there is no package called ‘targetscan.Hs.eg.db’
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘targetscan.Hs.eg.db’
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘targetscan.Hs.egMIRNA’
    mirna2targetscan: no visible binding for global variable
      ‘targetscan.Hs.egMIRBASE2FAMILY’
    mirna2targetscan: no visible binding for global variable
      ‘targetscan.Hs.egTARGETS’
    mirna2targetscan: no visible binding for global variable
      ‘targetscan.Hs.egTARGETSFULL’
    mirna2targetscan: no visible binding for global variable
      ‘targetscan.Mm.egMIRNA’
    mirna2targetscan: no visible binding for global variable
      ‘targetscan.Mm.egMIRBASE2FAMILY’
    mirna2targetscan: no visible binding for global variable
      ‘targetscan.Mm.egTARGETS’
    mirna2targetscan: no visible binding for global variable
      ‘targetscan.Mm.egTARGETSFULL’
    Undefined global functions or variables:
      as.tibble changes hits iso_sample pct targetscan.Hs.egMIRBASE2FAMILY
      targetscan.Hs.egMIRNA targetscan.Hs.egTARGETS
      targetscan.Hs.egTARGETSFULL targetscan.Mm.egMIRBASE2FAMILY
      targetscan.Mm.egMIRNA targetscan.Mm.egTARGETS
      targetscan.Mm.egTARGETSFULL total
    ```

# jmv

<details>

* Version: 1.2.5
* Source code: https://github.com/cran/jmv
* Date/Publication: 2020-02-17 16:40:14 UTC
* Number of recursive dependencies: 193

Run `revdep_details(,"jmv")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BayesFactor’ ‘GGally’ ‘GPArotation’ ‘MASS’ ‘PMCMR’ ‘R6’ ‘ROCR’
      ‘afex’ ‘car’ ‘emmeans’ ‘ggridges’ ‘lavaan’ ‘multcomp’ ‘mvnormtest’
      ‘nnet’ ‘psych’ ‘vcd’ ‘vcdExtra’
      All declared Imports should be used.
    ```

# jsmodule

<details>

* Version: 1.0.8
* Source code: https://github.com/cran/jsmodule
* URL: https://github.com/jinseob2kim/jsmodule
* BugReports: https://github.com/jinseob2kim/jsmodule/issues
* Date/Publication: 2020-06-04 17:00:02 UTC
* Number of recursive dependencies: 188

Run `revdep_details(,"jsmodule")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Cairo’ ‘survC1’
      All declared Imports should be used.
    ```

# LANDD

<details>

* Version: 1.1.0
* Source code: https://github.com/cran/LANDD
* Date/Publication: 2016-10-01 01:14:24
* Number of recursive dependencies: 103

Run `revdep_details(,"LANDD")` for more info

</details>

## In both

*   checking R code for possible problems ... NOTE
    ```
    getGO: no visible global function definition for ‘new’
    getGO: no visible global function definition for ‘getGeneric’
    graph.kd: no visible global function definition for ‘as’
    Undefined global functions or variables:
      as getGeneric new
    Consider adding
      importFrom("methods", "as", "getGeneric", "new")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# MAINT.Data

<details>

* Version: 2.3.1
* Source code: https://github.com/cran/MAINT.Data
* Date/Publication: 2020-01-07 11:10:03 UTC
* Number of recursive dependencies: 66

Run `revdep_details(,"MAINT.Data")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        libs   3.7Mb
    ```

# MAST

<details>

* Version: 1.14.0
* Source code: https://github.com/cran/MAST
* URL: https://github.com/RGLab/MAST/
* BugReports: https://github.com/RGLab/MAST/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 194

Run `revdep_details(,"MAST")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.5Mb
      sub-directories of 1Mb or more:
        data   4.1Mb
        doc    5.2Mb
    ```

# MCbiclust

<details>

* Version: 1.12.0
* Source code: https://github.com/cran/MCbiclust
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 177

Run `revdep_details(,"MCbiclust")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.6Mb
      sub-directories of 1Mb or more:
        data   3.1Mb
        doc    5.5Mb
    ```

# mgcViz

<details>

* Version: 0.1.6
* Source code: https://github.com/cran/mgcViz
* URL: https://github.com/mfasiolo/mgcViz
* BugReports: https://github.com/mfasiolo/mgcViz/issues
* Date/Publication: 2020-03-04 15:10:02 UTC
* Number of recursive dependencies: 95

Run `revdep_details(,"mgcViz")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        doc   4.4Mb
    ```

# MissingDataGUI

<details>

* Version: 0.2-5
* Source code: https://github.com/cran/MissingDataGUI
* Date/Publication: 2016-04-25 08:58:53
* Number of recursive dependencies: 102

Run `revdep_details(,"MissingDataGUI")` for more info

</details>

## In both

*   checking S3 generic/method consistency ... WARNING
    ```
    
    (R:49914): Gtk-WARNING **: gtk_disable_setlocale() must be called before gtk_init()
    See section ‘Generic functions and methods’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking replacement functions ... WARNING
    ```
    
    (R:50079): Gtk-WARNING **: gtk_disable_setlocale() must be called before gtk_init()
    The argument of a replacement function which corresponds to the right
    hand side must be named ‘value’.
    ```

*   checking for missing documentation entries ... WARNING
    ```
    
    (R:50734): Gtk-WARNING **: gtk_disable_setlocale() must be called before gtk_init()
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    
    (R:50780): Gtk-WARNING **: gtk_disable_setlocale() must be called before gtk_init()
    
    (R:50817): Gtk-WARNING **: gtk_disable_setlocale() must be called before gtk_init()
    
    (R:50949): Gtk-WARNING **: gtk_disable_setlocale() must be called before gtk_init()
    ```

*   checking dependencies in R code ... NOTE
    ```
    
    (R:49828): Gtk-WARNING **: gtk_disable_setlocale() must be called before gtk_init()
    ```

*   checking foreign function calls ... NOTE
    ```
    
    (R:50140): Gtk-WARNING **: gtk_disable_setlocale() must be called before gtk_init()
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    
    (R:50408): Gtk-WARNING **: gtk_disable_setlocale() must be called before gtk_init()
    ```

*   checking Rd \usage sections ... NOTE
    ```
    
    (R:51056): Gtk-WARNING **: gtk_disable_setlocale() must be called before gtk_init()
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# mlrMBO

<details>

* Version: 1.1.4
* Source code: https://github.com/cran/mlrMBO
* URL: https://github.com/mlr-org/mlrMBO
* BugReports: https://github.com/mlr-org/mlrMBO/issues
* Date/Publication: 2020-02-28 21:20:03 UTC
* Number of recursive dependencies: 131

Run `revdep_details(,"mlrMBO")` for more info

</details>

## In both

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘DiceOptim’
    ```

# MOFA

<details>

* Version: 1.4.0
* Source code: https://github.com/cran/MOFA
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 97

Run `revdep_details(,"MOFA")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘MOFA-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: DataOptions
    > ### Title: DataOptions: set and retrieve data options
    > ### Aliases: DataOptions DataOptions<- DataOptions,MOFAmodel-method
    > ###   DataOptions<-,MOFAmodel,list-method
    > 
    > ### ** Examples
    > 
    > # load a trained MOFAmodel object
    > filepath <- system.file("extdata", "scMT_model.hdf5", package = "MOFAdata")
    > MOFAobject <- loadModel(filepath)
    Error in h5checktypeOrOpenLoc(file, readonly = TRUE, native = native) : 
      Error in h5checktypeOrOpenLoc(). Cannot open file. File '' does not exist.
    Calls: loadModel -> h5read -> h5checktypeOrOpenLoc
    Execution halted
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘MOFAdata’
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        doc   4.6Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotEnrichmentDetailed: no visible binding for global variable
      ‘pathway’
    plotEnrichmentDetailed: no visible binding for global variable
      ‘feature.statistic’
    Undefined global functions or variables:
      feature.statistic pathway
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘PCGSE’
    ```

# NetworkChange

<details>

* Version: 0.6
* Source code: https://github.com/cran/NetworkChange
* Date/Publication: 2020-02-06 05:20:02 UTC
* Number of recursive dependencies: 135

Run `revdep_details(,"NetworkChange")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘sna’
      All declared Imports should be used.
    ```

# nzelect

<details>

* Version: 0.4.0
* Source code: https://github.com/cran/nzelect
* Date/Publication: 2017-10-02 20:35:23 UTC
* Number of recursive dependencies: 73

Run `revdep_details(,"nzelect")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        data   5.0Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 6409 marked UTF-8 strings
    ```

# ORFik

<details>

* Version: 1.8.1
* Source code: https://github.com/cran/ORFik
* URL: https://github.com/Roleren/ORFik
* BugReports: https://github.com/Roleren/ORFik/issues
* Date/Publication: 2020-04-29
* Number of recursive dependencies: 128

Run `revdep_details(,"ORFik")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘GenomicFeatures:::.merge_seqinfo_and_infer_missing_seqlengths’
      ‘IRanges:::regroupBySupergroup’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    windowCoveragePlot: no visible binding for global variable
      ‘fraction_min’
    windowCoveragePlot: no visible binding for global variable ‘position’
    Undefined global functions or variables:
      . CDS CDSGrouping Hx LEADERS ORFGrouping ORFScores RRS RSS
      StartCodons StopCodons TOP View chr codonSums count countRFP
      count_seq_pos_with_count dif difPer disengagementScores distORFCDS
      entropyRFP exon_rank feature forward fpkmRFP fpkmRNA fraction
      fraction.x fraction.y fractionLengths fraction_min fractions frame
      frame_one_RP frame_two_RP gene_sum grnames inFrameCDS ioScore
      isOverlappingCds kozak leaders mRNA median_score offsets_start ones
      position rankInTx ranks ratio_cds_leader ratio_cds_mrna
      ratio_mrna_aligned read.csv2 read.table rowSums2 scalingFactor seq1
      seq2 seq3 seq4 seq5 size startCodonCoverage startRegionRelative te
      trailers tx utr3_len utr5_len value variable widths windowMean
      windowSD write.csv write.table zscore
    Consider adding
      importFrom("graphics", "frame")
      importFrom("utils", "View", "read.csv2", "read.table", "write.csv",
                 "write.table")
    to your NAMESPACE file.
    ```

# PAFway

<details>

* Version: 0.1.3
* Source code: https://github.com/cran/PAFway
* Date/Publication: 2020-02-05 16:40:02 UTC
* Number of recursive dependencies: 75

Run `revdep_details(,"PAFway")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘network’ ‘scales’ ‘sna’
      All declared Imports should be used.
    ```

# plotly

<details>

* Version: 4.9.2.1
* Source code: https://github.com/cran/plotly
* URL: https://plotly-r.com, https://github.com/ropensci/plotly#readme, https://plot.ly/r
* BugReports: https://github.com/ropensci/plotly/issues
* Date/Publication: 2020-04-04 19:50:02 UTC
* Number of recursive dependencies: 150

Run `revdep_details(,"plotly")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        htmlwidgets   3.7Mb
    ```

# PopGenReport

<details>

* Version: 3.0.4
* Source code: https://github.com/cran/PopGenReport
* URL: https://github.com/green-striped-gecko/PopGenReport
* Date/Publication: 2019-02-04 12:13:23 UTC
* Number of recursive dependencies: 119

Run `revdep_details(,"PopGenReport")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘ecodist’
    ```

# pubh

<details>

* Version: 1.1.14
* Source code: https://github.com/cran/pubh
* URL: https://github.com/josie-athens/pubh
* BugReports: https://github.com/josie-athens/pubh/issues
* Date/Publication: 2020-06-02 06:50:02 UTC
* Number of recursive dependencies: 203

Run `revdep_details(,"pubh")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Hmisc’
      All declared Imports should be used.
    ```

# robCompositions

<details>

* Version: 2.2.1
* Source code: https://github.com/cran/robCompositions
* Date/Publication: 2020-02-11 17:20:02 UTC
* Number of recursive dependencies: 123

Run `revdep_details(,"robCompositions")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘mvoutlier’, ‘StatDA’
    ```

# robustbase

<details>

* Version: 0.93-6
* Source code: https://github.com/cran/robustbase
* URL: http://robustbase.r-forge.r-project.org/
* Date/Publication: 2020-03-23 17:40:02 UTC
* Number of recursive dependencies: 72

Run `revdep_details(,"robustbase")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘robustX’
    ```

# robustSingleCell

<details>

* Version: 0.1.1
* Source code: https://github.com/cran/robustSingleCell
* URL: https://github.com/asmagen/robustSingleCell
* BugReports: https://github.com/asmagen/robustSingleCell/issues
* Date/Publication: 2019-04-23 22:00:02 UTC
* Number of recursive dependencies: 149

Run `revdep_details(,"robustSingleCell")` for more info

</details>

## Newly broken

*   checking whether package ‘robustSingleCell’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘GGally::mean_sd’ by ‘ggpubr::mean_sd’ when loading ‘robustSingleCell’
      Warning: replacing previous import ‘GGally::median_iqr’ by ‘ggpubr::median_iqr’ when loading ‘robustSingleCell’
    ```

# rrr

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/rrr
* URL: http://github.com/chrisaddy/rrr
* Date/Publication: 2016-12-09 15:15:55
* Number of recursive dependencies: 85

Run `revdep_details(,"rrr")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rcpp’
      All declared Imports should be used.
    ```

# scPipe

<details>

* Version: 1.10.0
* Source code: https://github.com/cran/scPipe
* URL: https://github.com/LuyiTian/scPipe
* BugReports: https://github.com/LuyiTian/scPipe
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 137

Run `revdep_details(,"scPipe")` for more info

</details>

## In both

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘scater’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    anno_to_saf: no visible binding for global variable ‘GeneID’
    convert_geneid: no visible global function definition for ‘useMart’
    get_genes_by_GO: no visible global function definition for ‘useMart’
    infer_gene_id_from_parent: no visible binding for global variable
      ‘type’
    plot_demultiplex: no visible binding for global variable ‘status’
    plot_demultiplex: no visible binding for global variable ‘count’
    plot_demultiplex: no visible binding for global variable ‘label_y’
    plot_demultiplex: no visible binding for global variable ‘label_tx’
    Undefined global functions or variables:
      GeneID count label_tx label_y status type useMart
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# SeqSQC

<details>

* Version: 1.10.0
* Source code: https://github.com/cran/SeqSQC
* URL: https://github.com/Liubuntu/SeqSQC
* BugReports: https://github.com/Liubuntu/SeqSQC/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 125

Run `revdep_details(,"SeqSQC")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        doc       1.9Mb
        extdata   3.3Mb
    ```

# SmartEDA

<details>

* Version: 0.3.5
* Source code: https://github.com/cran/SmartEDA
* URL: https://daya6489.github.io/SmartEDA/
* BugReports: https://github.com/daya6489/SmartEDA/issues
* Date/Publication: 2020-04-25 06:20:02 UTC
* Number of recursive dependencies: 112

Run `revdep_details(,"SmartEDA")` for more info

</details>

## In both

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘InformationValue’, ‘DataExplorer’
    ```

# spinifex

<details>

* Version: 0.1.0
* Source code: https://github.com/cran/spinifex
* URL: https://github.com/nspyrison/spinifex/
* BugReports: https://github.com/nspyrison/spinifex/issues
* Date/Publication: 2019-04-09 12:00:04 UTC
* Number of recursive dependencies: 89

Run `revdep_details(,"spinifex")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘GGally’ ‘tibble’ ‘webshot’
      All declared Imports should be used.
    ```

# spup

<details>

* Version: 1.3-2
* Source code: https://github.com/cran/spup
* Date/Publication: 2020-04-30 22:20:06 UTC
* Number of recursive dependencies: 78

Run `revdep_details(,"spup")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘sp’
      All declared Imports should be used.
    ```

# statVisual

<details>

* Version: 1.2.1
* Source code: https://github.com/cran/statVisual
* Date/Publication: 2020-02-20 19:30:02 UTC
* Number of recursive dependencies: 161

Run `revdep_details(,"statVisual")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gbm’ ‘ggfortify’ ‘tibble’ ‘tidyverse’
      All declared Imports should be used.
    ```

# tidybulk

<details>

* Version: 1.0.0
* Source code: https://github.com/cran/tidybulk
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 245

Run `revdep_details(,"tidybulk")` for more info

</details>

## In both

*   checking examples ... ERROR
    ```
    ...
    ℹ Input `n` is `n()`.
    ℹ The error occured in group 1: vs = 0, am = 0.
    Backtrace:
         █
      1. └─by_vs_am %>% summarise(n = n())
      2.   ├─base::withVisible(eval(quote(`_fseq`(`_lhs`)), env, env))
      3.   └─base::eval(quote(`_fseq`(`_lhs`)), env, env)
      4.     └─base::eval(quote(`_fseq`(`_lhs`)), env, env)
      5.       └─`_fseq`(`_lhs`)
      6.         └─magrittr::freduce(value, `_function_list`)
      7.           ├─base::withVisible(function_list[[k]](value))
      8.           └─function_list[[k]](value)
      9.             ├─tidybulk::summarise(., n = n())
     10.             └─tidybulk:::summarise.default(., n = n())
     11.               ├─dplyr::summarise(.data, ...)
     12.               └─dplyr:::summarise.grouped_df(.data, ...)
     13.                 └─dplyr:::summarise_cols(.data, ...)
    <parent: error/simpleError>
    Backtrace:
    █
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
        `Fraction of variance`    PC
                         <dbl> <int>
      1                0.990       1
      2                0.00310     2
      Getting the 100 most variable genes
      Getting the 5 most variable genes
      Getting the 5 most variable genes
      ══ testthat results  ═══════════════════════════════════════════════════════════
      [ OK: 139 | SKIPPED: 0 | WARNINGS: 18 | FAILED: 3 ]
      1. Error: Only scaled counts - no object (@test-bulk_methods.R#65) 
      2. Error: tidybulk SummarizedExperiment normalisation manual (@test-bulk_methods_SummarizedExperiment.R#29) 
      3. Error: tidybulk SummarizedExperiment normalisation (@test-bulk_methods_SummarizedExperiment.R#50) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.4Mb
      sub-directories of 1Mb or more:
        data   7.1Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Missing object imported by a ':::' call: ‘dplyr:::flatten_bindable’
    package 'methods' is used but not declared
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    tidybulk,RangedSummarizedExperiment: no visible binding for global
      variable ‘.’
    tidybulk,RangedSummarizedExperiment: no visible binding for global
      variable ‘feature’
    tidybulk,SummarizedExperiment: no visible binding for global variable
      ‘.’
    tidybulk,SummarizedExperiment: no visible binding for global variable
      ‘feature’
    Undefined global functions or variables:
      (Intercept) . .a .abundance_scaled GeneID Status X_cibersort Y assay
      buildIdx cluster cluster kmeans cmdscale.out correlation counts
      data_base egsea egsea.base ensembl_id entrez
      error_if_parameters_not_match feature genes item1 lowly_abundant m
      med med.rank multiplier my_n n_aggr name pathway rc read count
      ref_genome rotated dimensions rotation sample 1 sample 2 sample a
      sample b sample_idx samples sdev seurat_clusters temp tot tot_filt
      transcript tt_columns value variable x
    Consider adding
      importFrom("base", "sample")
      importFrom("stats", "kmeans")
    to your NAMESPACE file.
    ```

# TVTB

<details>

* Version: 1.14.0
* Source code: https://github.com/cran/TVTB
* URL: https://github.com/kevinrue/TVTB
* BugReports: https://github.com/kevinrue/TVTB/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 158

Run `revdep_details(,"TVTB")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        R     2.0Mb
        doc   2.7Mb
    ```

# ubiquity

<details>

* Version: 1.0.1
* Source code: https://github.com/cran/ubiquity
* URL: https://r.ubiquity.tools/
* BugReports: https://github.com/john-harrold/ubiquity/issues
* Date/Publication: 2020-01-21 07:50:02 UTC
* Number of recursive dependencies: 104

Run `revdep_details(,"ubiquity")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  9.3Mb
      sub-directories of 1Mb or more:
        doc     6.4Mb
        ubinc   2.0Mb
    ```

# ufs

<details>

* Version: 0.3.1
* Source code: https://github.com/cran/ufs
* URL: https://r-packages.gitlab.io/ufs
* BugReports: https://gitlab.com/r-packages/ufs/issues
* Date/Publication: 2019-08-22 17:30:02 UTC
* Number of recursive dependencies: 116

Run `revdep_details(,"ufs")` for more info

</details>

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘viridis’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘userfriendlyscience’, ‘behaviorchange’
    ```

# vidger

<details>

* Version: 1.8.0
* Source code: https://github.com/cran/vidger
* URL: https://github.com/btmonier/vidger, https://bioconductor.org/packages/release/bioc/html/vidger.html
* BugReports: https://github.com/btmonier/vidger/issues
* Date/Publication: 2020-04-27
* Number of recursive dependencies: 113

Run `revdep_details(,"vidger")` for more info

</details>

## In both

*   checking installed package size ... NOTE
    ```
      installed size is 11.2Mb
      sub-directories of 1Mb or more:
        data   4.7Mb
        doc    6.1Mb
    ```

